app/views/users/mfa_selection/index.html.erb

```diff
@@ -2,12 +2,6 @@
 
 <%= render PageHeadingComponent.new.with_content(t('two_factor_authentication.two_factor_choice')) %>
 
-<% if IdentityConfig.store.select_multiple_mfa_options %>
```

Comment:
We still need this to be under a feature toggle. It should be that we have two texts depending on if the feature toggle is turned on or off.